src/main/java/pl/futurecollars/invoicing/db/memory/InMemoryDatabase.java

```diff
@@ -5,14 +5,10 @@
 import java.util.List;
 import java.util.Optional;
 import lombok.Data;
-import org.springframework.context.annotation.Primary;
-import org.springframework.stereotype.Repository;
 import pl.futurecollars.invoicing.db.Database;
 import pl.futurecollars.invoicing.model.Invoice;
 
-@Repository
 @Data
```

Comment:
to @Data jest przydatne na klasach modelu danych - tutaj generuje nam dodatkowo dużo niepotrzebnych metod - zastanów się czy tutaj jakas jest potrzebna